includes/class.llms.template.loader.php

```diff
@@ -437,23 +459,39 @@ public function template_loader( $template ) {
 
 			$template = 'archive-llms_membership.php';
 
-		} elseif ( is_single() && ( get_post_type() == 'llms_certificate' || get_post_type() == 'llms_my_certificate' ) ) {
+		} elseif ( is_single() && ( get_post_type() === 'llms_certificate' || get_post_type() === 'llms_my_certificate' ) ) {
 
 			$template = 'single-certificate.php';
 
 		} else {
 
 			return $template;
 
-		} // End if().
+		}
 
-		// check for an override file
+		// check for an override file.
 		$override      = llms_get_template_override( $template );
 		$template_path = $override ? $override : LLMS()->plugin_path() . '/templates/';
 		return $template_path . $template;
 
 	}
 
+	/**
+	 * Maybe print notices after redirection.
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	private function maybe_print_notices_on_sales_page_redirect() {
+
+		if ( ! empty( $_GET['llms_print_notices'] ) ) { // phpcs:ignore -- No need to verify nonce for this purpose.
```

Comment:
Use `llms_filter_input()` and you shouldn't run into nonce verification warnings.

And additionally when disabling phpcs on a single line it's preferable to disable phpcs for the specific rule instead of entirely. EG:

`// phpcs:disable WordPress.Security.NonceVerification.Missing`

This way if we happen to encounter other violations on the same line we can clean those up!

Reply:
Of course you're right (on both).
I usually tend to disable `phpcs` for the single rule but this time I didn't, for no reason, I just did it wrong... sorry
I'm going to fix it tomorrow (I'm from my bed now 😅).

Reply:
No worries man!

Reply:
done in cfc8e6e